Jumpscale/builder/system/BuilderBaseClass.py

```diff
@@ -113,9 +113,10 @@ def flist_create(self, sandbox_dir=None, hub_instance=None):
             j.builder.tools.file_ensure(file_dest)
             j.builder.tools.file_write(file_dest, self.startup)
 
-        ld_dest = j.sal.fs.joinPaths(sandbox_dir, 'lib64/')
-        j.builder.tools.dir_ensure(ld_dest)
-        j.sal.fs.copyFile('/lib64/ld-linux-x86-64.so.2', ld_dest)
+        if j.core.platformtype.myplatform.isLinux:
```

Comment:
but we will still need the flist to contain this file even if it wasn't built on OSX

Reply:
How will you add it to the flist if it is not present on MacOSX ?

Reply:
@andrewayoub Why do we need this file ? Is it a requirement to run this flist in a container on zero-os ?

Reply:
yes the container won't work without this file (we knew  that by trying so I'm not very sure) maybe there is an alternative or we can download this file from somewhere, I'll ask Azmi

Reply:
The problem now is that the flists will run on a linux os, which means that even if your binaries are statically built  (and you won't need this dynamic builder)  it should also be cross compiled for linux, because it has to have the linux ELF, so I'd say it will be very complicated to create flists from an osx machine

Reply:
Can we remove this commit from the pull request and move it to a different discussion? This way the other fixes can be merged.

Reply:
This is moved to a seperate branch: https://github.com/threefoldtech/jumpscaleX/tree/fix/golang_mac